### ISSUE TITLE
Visit HTTP API feature requirements like visit cli

### ISSUE THREAD

**Issue by akhilambati** [vespa-engine/vespa#25384]
**Is your feature request related to a problem? Please describe.**
 its a requirement in visit http api to have functionality similar to visit cli. As visit http api has higher throught, but visit http api!=visit cli. can we please expect an option in visit http api to take from and to timestamp and also returns the progress (for each slice if slicing is used), so that we can retrieve docs added/updated and doc ids for deleted between that from and to timestamp

**Describe the solution you'd like**
Basically i am using visit for incremental hourly backup, As I have two options using visit cli and visit http api, knowing that visit http api has higher troughput giving the option for slicing and stream, requesting for an option as mentioned above for http api

**Comment by jonmv** [comment]
#28439 adds the `includeRemoves` HTTP parameter to visit remove operations, which will add entries `{ "remove": "id:my:doc::id" }` to the "documents" array of the HTTP JSON response. From and to timestamps were added earlier. I'll update the documentation as well when the API change is ratified.
